chainerx_cc/chainerx/cuda/cuda_device.h

```diff
@@ -213,14 +241,9 @@ class CudaDevice : public Device {
     std::shared_ptr<MemoryPool> pinned_memory_pool_;
 
     // Memory keeper.
-    cuda_internal::MemoryKeeper memory_keeper_{};
+    cuda_internal::DeviceInternals device_internals_;
```

Comment:
Incorrect correspondence with a comment.

Reply:
Fixed in https://github.com/chainer/chainer/pull/6827.